### DISCUSSION TITLE
Beta 14.0 (23A5286g) is working!

### DISCUSSION THREAD

**Discussion by NikitinVit** [daliansky/XiaoMi-Pro#743] [General]
Beta 14.0 (23A5286g) is working! Computer is reboot while trying to install the macOS 14.0 (23A344) Sonoma release!

**Comment by stevezz3** [comment]
Could you try the latest OC release posted in https://github.com/daliansky/XiaoMi-Pro-Hackintosh/releases/tag/v1.8.0, which includes the latest AirportItlwm.
It seems like Apple changes the network stack kernel again in recent version, according to https://github.com/OpenIntelWireless/itlwm/issues/883#issuecomment-1670749680

**Reply by NikitinVit** [reply]
Beta 14.0 (23A5286g) is working! Wi-Fi and Bluetooth also work on XiaoMi NoteBook Pro EFI v1.7.9 version

**Reply by NikitinVit** [reply]
Beta 14.0 (23A5286g) https://techrechard.com/download-macos-sonoma-dmg-file/

**Comment by NikitinVit** [comment]
macOS 14.0 (23A344) is working! there was an error in kext AirportItlwm_Sonoma.kext! downloaded the new version works without reboots
Wi-Fi and Bluetooth work on XiaoMi NoteBook Pro EFI v1.8.0 build! macOS 14.0 (23A344) is working!

**Reply by NoudF** [reply]
I installed Sonoma 14.0 today and used the EFI 1.8.0 OC zip. The system boots fine and WiFi and Bluetooth both work, but I cannot use the microphone of a bluetooth device.
If I try to do so, Bluetooth crashes and the device disconnect.
Is this a common issue, or does this not happen for you?
And if not, can you help me fix the issue, by pointing me in the right direction regarding files to download.

**Reply by stevezz3** [reply]
For any Intel Bluetooth issue, you have to report to https://github.com/OpenIntelWireless/IntelBluetoothFirmware/issues for additional help. The only thing I can make sure is that the driver in the EFI release pack is up-to-date.

**Reply by NoudF** [reply]
Thanks for the response! I downloaded the latest alpha build from there and replaced the one in the kexts folder. After a reboot it now works, both audio and mic.
